[2022-11-13T17:12:34.390+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v1.insert_into_table scheduled__2022-11-09T00:00:00+00:00 [queued]>
[2022-11-13T17:12:34.409+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v1.insert_into_table scheduled__2022-11-09T00:00:00+00:00 [queued]>
[2022-11-13T17:12:34.411+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-13T17:12:34.414+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2022-11-13T17:12:34.417+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-13T17:12:34.456+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-11-09 00:00:00+00:00
[2022-11-13T17:12:34.472+0000] {standard_task_runner.py:55} INFO - Started process 3426 to run task
[2022-11-13T17:12:34.478+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v1', 'insert_into_table', 'scheduled__2022-11-09T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpgm03qedw']
[2022-11-13T17:12:34.482+0000] {standard_task_runner.py:83} INFO - Job 19: Subtask insert_into_table
[2022-11-13T17:12:34.587+0000] {task_command.py:376} INFO - Running <TaskInstance: dag_with_postgres_operator_v1.insert_into_table scheduled__2022-11-09T00:00:00+00:00 [running]> on host 6a6c6e07e37b
[2022-11-13T17:12:34.709+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=charlie
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v1
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-11-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-09T00:00:00+00:00
[2022-11-13T17:12:34.728+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-11-13T17:12:34.748+0000] {sql.py:315} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-11-09', 'dag_with_postgres_operator_v1')
        , parameters: None
[2022-11-13T17:12:34.751+0000] {sql.py:324} INFO - Rows affected: 1
[2022-11-13T17:12:34.771+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v1, task_id=insert_into_table, execution_date=20221109T000000, start_date=20221113T171234, end_date=20221113T171234
[2022-11-13T17:12:34.830+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-13T17:12:34.913+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
